cypress/utils/storeFront/storeFrontProductUtils.js

```diff
@@ -0,0 +1,34 @@
+import ProductDetails from "../../apiRequests/storeFront/ProductDetails";
+import Search from "../../apiRequests/storeFront/Search";
+
+class StoreFrontProductUtils {
```

Comment:
Creating a class only makes sense if you do something with its instance and operate on a set of internal (instance) values. Here you are defining methods that do not operate on any instance-scoped values. Instead of making a single export class, you can get rid off the class and export these 3 individually:
```
export function isProductVisible(...)
export function isProductAvailableForPurchase(...)
export function isProductVisibleInSearchResult(...)
```
then in the file you want to use them, simply:
```
import { isProductVisible } from "[...]/utils/storeFront/storeFrontProductUtils";
```